packages/airnode-admin/src/sdk.ts

```diff
@@ -48,49 +52,62 @@ export class AdminSdk {
   sponsorToWithdrawalRequestCount = (sponsorAddress: string) =>
     admin.sponsorToWithdrawalRequestCount(this.airnodeRrp, sponsorAddress);
 
-  fulfillWithdrawal = (requestId: string, airnodeAddress: string, sponsorAddress: string, amount: string) =>
-    admin.fulfillWithdrawal(this.airnodeRrp, requestId, airnodeAddress, sponsorAddress, amount);
+  // fulfillWithdrawal = (requestId: string, airnodeAddress: string, sponsorAddress: string, amount: string) =>
+  //   admin.fulfillWithdrawal(this.airnodeRrp, requestId, airnodeAddress, sponsorAddress, amount);
+  // TODO: check overrides here that amount (i.e. overrides.value) is passed on correctly
```

Comment:
The implementation looks good. Or is there something I am missing from this comment?

Reply:
I had forgotten to remove this